test/functional/search/search-sort.bats

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+global_setup() {
+
+	create_test_environment "$TEST_NAME"
+
+	small=/small-file:"$(create_file "$WEBDIR"/10/files 1MB)"
+	medium=/medium-file:"$(create_file "$WEBDIR"/10/files 2MB)"
+	large=/large-file:"$(create_file "$WEBDIR"/10/files 3MB)"
+
+	create_bundle -n alfa-zulu -f /yankee/alfalfa,/alfaromeo/echo,/foo/quebec,/foo/alfa,/foo/delta,/bar/alfa,"$small" "$TEST_NAME"
+	create_bundle -n victor -f /yankee,/bar/alfa,/alfa/zulu,/papa,"$large" "$TEST_NAME"
+	create_bundle -n alfa-charly -f /foxtrot,/tango/sierra,/kilo,"$medium" "$TEST_NAME"
+
+}
+
+test_setup() {
+
+	# do nothing
+	return
+
+}
+
+test_teardown() {
+
+	# do nothing
+	return
+
+}
+
+global_teardown() {
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+@test "SRH023: search shows the results in alphabetical order" {
+
+	run sudo sh -c "$SWUPD search-legacy $SWUPD_OPTS -T 10 alfa"
+
+	assert_status_is 0
+	expected_output=$(cat <<-EOM
+		Bundle alfa-charly	\\(2 MB to install\\)
+		./usr/share/clear/bundles/alfa-charly
+		Bundle alfa-zulu	\\(1 MB to install\\)
+		./alfaromeo/echo
+		./bar/alfa
+		./foo/alfa
+		./usr/share/clear/bundles/alfa-zulu
+		./yankee/alfalfa
+		Bundle victor	\\(3 MB to install\\)
```

Comment:
shouldn't we order bundle names too?

Reply:
what do you mean? bundle names are also ordered alphabetically:
Bundle alfa-charly
Bundle alfa-zulu
Bundle victor

Reply:
hahhaa. I got confused when I read the bundle names. You are right.